### PR TITLE
Add PR template

### PR DESCRIPTION
**Pull request by zhilingc**
Adapted from the kubernetes PR template.

### CODE REVIEW

**Comment by davidheryanto** [comment]
Can we put a reference to an example release note if we are following the Kubernetes's ones?
Screenshot from 2020-01-08 13-31-21
I think it will be clearer for people if there is an example.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+
+1. Ensure that your code follows our code conventions: https://github.com/gojek/feast/blob/master/docs/contributing.md#code-conventions
+2. Run unit tests and ensure that they are passing: https://github.com/gojek/feast/blob/master/docs/contributing.md#running-unit-tests
+3. If your change introduces any API changes, make sure to update the integration tests scripts here: https://github.com/gojek/feast/tree/master/tests/e2e
+4. Make sure documentation is updated for your PR!
```
Comment (woop) [inline .github/pull_request_template.md]: Make sure you have signed the CLA https://cla.developers.google.com/clas

Comment (zhilingc) [reply .github/pull_request_template.md]: added

**Comment by zhilingc** [comment]
> Can we put a reference to an example release note if we are following the Kubernetes's ones?
> Screenshot from 2020-01-08 13-31-21
> I think it will be clearer for people if there is an example.

done

**Pull request merged**